apps/authorization/tests/test_data_access_grant_permissions.py

```diff
@@ -265,7 +265,7 @@ def _assert_call_token_refresh_endpoint(
             "client_id": application.client_id,
             "client_secret": application.client_secret_plain,
         }
-        response = self.client.post("/v1/o/token/", data=refresh_post_data)
+        response = self.client.post("/v2/o/token/", data=refresh_post_data)
```

Comment:
Although trivial, lets use this opportunity to update these kinds of things to the `Versions` constants if we're going to touch them?